src/components/OrderItemRejHistoryModal.vue

```diff
@@ -92,7 +92,8 @@ export default defineComponent({
       modalController.dismiss({ dismissed: true });
     },
     getRejectReasonDescription(rejectReasonEnumId: string) {
-      return this.rejectReasons.find((reason: any) => reason.enumId === rejectReasonEnumId)?.description;
+      const reason = this.rejectReasons.find((reason: any) => reason.enumId === rejectReasonEnumId)
+      return reason?.description ? reason.description : reason?.enumDescription;
```

Comment:
These checks might lead to an incorrect description. My understanding is that we’re fetching data from two different view entities:

For Admin users: EnumTypeChildAndEnum
For other users: EnumerationGroupAndMember

In the case of an Admin user, the current logic works as expected. However, for other users, it may not work correctly because the EnumerationGroupAndMember view contains both description and enumDescription fields. Here, description refers to EnumGroup.description, while enumDescription refers to Enumeration.description.

Let me know if I am thinking wrong.

Reply:
This will not give issue, since we are not fetching description field in case of EnumerationGroupAndMember, so we won't have description and EnumDescription at the same time.

Reply:
This will not give issue, since we are not fetching description field in case of EnumerationGroupAndMember, so we won't have description and EnumDescription at the same time.